server/container_create.go

```diff
@@ -62,6 +64,33 @@ func addOciBindMounts(sb *sandbox, containerConfig *pb.ContainerConfig, specgen
 	return nil
 }
 
+func addDevices(sb *sandbox, containerConfig *pb.ContainerConfig, specgen *generate.Generator) error {
+	sp := specgen.Spec()
+	for _, device := range containerConfig.GetDevices() {
+		d, err := devices.DeviceFromPath(device.HostPath, device.Permissions)
+		if err != nil {
+			return err
```

Comment:
Yes we should log some kind of error here.  Unless DeviceFromPath logs something?